### PR TITLE
Refactoring grids filtering topics

### PR DESCRIPTION
**Pull request by HristoP96**
Closes this [issue](https://github.com/IgniteUI/trial-watermark/issues/67)

### CODE REVIEW

File: en/components/general/angular-grid-overview-guide.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Angular data grids support easy sorting, filtering, and pagination. With rich AP
 
 The Grid provides three types of Filtering with custom filtering conditions: 
 
- - [Filter row](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/filtering.html#grid-filtering-overview "Grid filtering Overview topic") per column with default filtering strategy provided out of the box, as well as all the standard filtering conditions. 
+ - [Filter row](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/filtering.html "Grid filtering Overview topic") per column with default filtering strategy provided out of the box, as well as all the standard filtering conditions.
```
Comment (dobromirts) [inline en/components/general/angular-grid-overview-guide.md:194]:
```suggestion
 - [Filter row](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/filtering.html "Grid Filtering Overview topic") per column with default filtering strategy provided out of the box, as well as all the standard filtering conditions. 
```

File: en/components/grids_templates/filtering.md [hunk 2]
```diff
@@ -20,11 +20,12 @@ _keywords: angular filter, ignite ui for angular, infragistics
...
 
-# @@igComponent Quick Filtering Overview
+# Angular @@igComponent Filtering
+_IgniteUI for Angular @@igComponent component provides three different filtering types - Quick filtering, [Excel style filtering](excel_style_filtering.md) and [Advanced filtering](advanced_filtering.md) which enable you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound._
 
-Angular grid filtering enables you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
 
-### Demo
+
+## Demo
```
Comment (dobromirts) [inline en/components/grids_templates/filtering.md:28]:
```suggestion
## Angular Filter Example
```

**Review by dobromirts — APPROVED** [review]

**Review by dobromirts — APPROVED** [review]

File: en/components/grids_templates/filtering.md [hunk 3]
```diff
@@ -20,11 +20,12 @@ _keywords: angular filter, ignite ui for angular, infragistics
 ---
 }
 
-# @@igComponent Quick Filtering Overview
+# Angular @@igComponent Filtering
+IgniteUI for Angular @@igComponent component provides three different filtering types - Quick filtering, [Excel style filtering](excel_style_filtering.md) and [Advanced filtering](advanced_filtering.md) which enable you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
```
Comment (zdrawku) [inline en/components/grids_templates/filtering.md:24]:
```suggestion
IgniteUI for Angular @@igComponent component provides three different filtering types - Quick filtering, [Excel style filtering](excel_style_filtering.md) and [Advanced filtering](advanced_filtering.md) which enable you to display only the records that meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
```

File: en/components/grids_templates/filtering.md [hunk 4]
```diff
@@ -20,11 +20,12 @@ _keywords: angular filter, ignite ui for angular, infragistics
...
 
-Angular grid filtering enables you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
 
-### Demo
+
+## Angular Filter Example
```
Comment (zdrawku) [inline en/components/grids_templates/filtering.md:29]: Can you add a brief description of the example - let's establish that as a pattern for all first demo examples for each grid topic

**Review by zdrawku — CHANGES_REQUESTED** [review]

File: en/components/grids_templates/filtering.md (hunk 4 above)
Comment (HristoP96) [reply en/components/grids_templates/filtering.md:29]: Done :)

**Review by zdrawku — APPROVED** [review]

**Pull request merged**